app/src/main/java/org/wikipedia/feed/onthisday/OnThisDay.kt

```diff
@@ -17,8 +17,8 @@ class OnThisDay {
 
     @Serializable
     class Event {
```

Comment:
Can we keep this class immutable? (move vals into constructor, and re-construct if changes are needed)

Reply:
Yes, done!